haystack/nodes/retriever/multimodal/embedder.py

```diff
@@ -3,21 +3,25 @@
 import logging
 from pathlib import Path
 
-import torch
 from tqdm.auto import tqdm
 import numpy as np
 from PIL import Image
```

Comment:
Let's double check whether this needs a LazyImport too. I'm guessing it does.

Reply:
For now I'd just install it, Pillow is not huge. But in the spirit of making all dependencies somewhat optional in the future, we could move it out as well. Let's do it in another PR.